algorithm/cgal/cutSurfaceMeshes.js

```diff
@@ -9,7 +9,7 @@ export const cutSurfaceMeshes = (targetMeshes, targetSegments, sources) => {
     const cutMeshes = [];
     const cutSegments = [];
     let nthSegments;
-    const status = getCgal().CutSurfaceMeshes(
+    const status = getCgal().ParallelCutSurfaceMeshes(
```

Comment:
Or no it is used. Is this parallel done using workers? If so where is the worker called from?

Reply:
It's a parallel algorithm, but in this case we're running the parallel components sequentially since threading is disabled.